packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -115,7 +115,15 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
     private async _create(id: CoreId, content: Request, peer: CoreAddress) {
         const canCreateResult = await this.canCreate({ content, peer });
 
-        if (canCreateResult.isError()) throw canCreateResult.error;
+        if (canCreateResult.isError()) {
+            if (canCreateResult.error.code === "error.consumption.validation.inheritedFromItem") {
+                throw ConsumptionCoreErrors.requests.inheritedFromItem(
+                    "Some child items have errors. If this error occurred during the specification of a Request, call 'canCreate' to get more information."
+                );
+            }
```

Comment:
```suggestion
            const error = ConsumptionCoreErrors.requests.inheritedFromItem("Some child items have errors. Call 'canCreate' to get more information.");
            if (canCreateResult.error.equals(error)) throw error;
```

I'd prefer to make this bulletproof and not to hardcode this error code.

Reply:
There is the problem that I want that when the error is  ConsumptionCoreErrors.requests.inheritedFromItem("Some child items have errors.") that I want to change it to  ConsumptionCoreErrors.requests.inheritedFromItem("Some child items have errors.  Call 'canCreate' to get more information.").

Reply:
I actually do exactly the same. `CoreError#equals` only compares the error code.